health/api/src/main/java/com/cerner/beadledom/health/dto/BuildDto.java

```diff
@@ -20,6 +19,8 @@
 @AutoValue
 @JsonDeserialize(builder = BuildDto.Builder.class)
 @ApiModel(description = "Indicates the list of dependencies of this service.")
+@io.swagger.annotations.ApiModel(
+    description = "Indicates the list of dependencies of this service.")
```

Comment:
Message needs to be updated.

Reply:
https://github.com/cerner/beadledom/pull/119/commits/6cc9839dbbd326009806d5ebe61c225f6a3a295b#diff-2e19244b40eaaaeb94e87b27b8f6b18fL21